docs/basics/functions.md

```diff
@@ -917,6 +918,97 @@ Which would return the something like following:
  }]
 ```
 
+**Example 2 (split_array and leaf_array)**
+
+Consider a service that returns the following payload:
+```json
+{
+    "concurrent_plays": {
+        "timestamps": [
+            1585662957000,
+            1585662958000,
+            1585662959000
+        ],
+        "meta": {
+            "status": 0
+        },
+        "type": "time_series",
+        "filters": {
+            "AccountA": [
+                200,
+                190,
+                180
+            ]
+        }
+    }
+}
+```
+
+You could split the leaf nodes within `timestamps` and `filters` branches into separate samples, then merge and join them together.
+
+```yaml
+name: leafArrayExample
+
+apis:
+  - name: getTimetamps
+    url: http://127.0.0.1:8887/concurrent_plays.json
+    split_array: true
+    leaf_array: true
+    set_header:  [Timestamp]
+    start_key:
+      - concurrent_plays
+      - timestamps
+    merge: myMetricsSample
+    join_key: index
+
+  - name: getValues
+    url: http://127.0.0.1:8887/concurrent_plays.json
+    split_array: true
+    leaf_array: true
+    set_header:  [Value]
+    start_key:
+      - concurrent_plays
+      - filters
+    custom_attributes:
+      accountid: "AccountA"
+    merge: myMetricsSample
+    join_key: index
+
+```
+Which would return the something like following:
```

Comment:
s/return the something like/return the following

Reply:
Updated functions.md accordingly.